fitnessllm_dataplatform/stream/strava/entities/queries.py

```
@@ -0,0 +1,30 @@
+    """
+
+
+@beartype
+def create_get_latest_activity_date_query(
+        athlete_id: str
+) -> str:
+    return f"""
+        SELECT MAX(start_date)
+        FROM dev_strava.activity
+        WHERE athlete_id = '{athlete_id}'
+    """
```

Ruff (0.8.2)

Possible SQL injection vector through string-based query construction

(S608)